src/commands/run.js

```diff
@@ -82,19 +85,41 @@ class RunCommand extends ApifyCommand {
             warning('You are not logged in with your Apify Account. Some features like Apify Proxy will not work. Call "apify login" to fix that.');
         }
 
-        // --max-http-header-size=80000
-        // Increases default size of headers. The original limit was 80kb, but from node 10+ they decided to lower it to 8kb.
-        // However they did not think about all the sites there with large headers,
-        // so we put back the old limit of 80kb, which seems to work just fine.
-        const currentNodeVersion = process.versions.node;
-        const lastSupportedVersion = semver.minVersion(SUPPORTED_NODEJS_VERSION);
-        if (semver.gte(currentNodeVersion, lastSupportedVersion)) {
-            env.NODE_OPTIONS = env.NODE_OPTIONS ? `${env.NODE_OPTIONS} --max-http-header-size=80000` : '--max-http-header-size=80000';
-        } else {
-            warning(`You are running Node.js version ${currentNodeVersion}, which is no longer supported. `
-                + `Please upgrade to Node.js version ${lastSupportedVersion} or later.`);
+        if (packageJsonExists) { // Actor is written in Node.js
+            const serverJsFile = path.join(cwd, 'server.js');
```

Comment:
The `server.js` file? I just mover this over from the previous version of the code, but I believe it's there to completely mimick the `npm start` behavior.
When you run `npm start`, and there's no `start` script, `npm` tries to run `node server.js`, so that's why we check for it here.

Reply:
Oh, sorry, in the diff it was in the "paragraph" above, so I didn't notice it. Didn't know that Node does this, thanks for explaining